src/features/embalm/stepContent/steps/FundBundlr.tsx

```diff
@@ -12,16 +13,14 @@ export function FundBundlr() {
         mb={6}
         variant="secondary"
       >
-        Bundlr is responsible for uploading your file payload to Arweave. You must have enough
-        Bundlr balance to upload the payload. You can connect to Bundlr and fund your account below
-        or on the Bundlr page in the main menu.
+        Bundlr will package your payload and send to Arweave using Ethereum.
```

Comment:
What is the reason for changing this text? If it's to make it shorter, we should maybe update the "using Ethereum" part, as it makes it sounds like Bundlr is using the Ethereum network to send the data to arweave. Could just truncate "using Ethereum" here, or update to a new sentence "Deposits to Bundlr will be made with ETH".

Reply:
Updated text came from designs. But yes I agree on the confusion there. I'll add the ETH deposit notice and remove 'using Ethereum'.